pkg/didcomm/crypto/jwe/authcrypt/decrypt.go

```diff
@@ -0,0 +1,40 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package authcrypt
+
+import (
+	"golang.org/x/crypto/blake2b"
+	"golang.org/x/crypto/chacha20poly1305"
+	"golang.org/x/crypto/nacl/box"
+)
+
+// Decrypt will JWE decode the envelop argument for the sender and recipients
+// Using (X)Chacha20 encryption algorithm and Poly1035 authenticator
+func (c *Crypter) Decrypt(envelope []byte) (string, error) {
+	decryptOID(nil, nil, nil)
+	return "", nil
+}
+
+// decryptOID will decrypt a recipient's encrypted OID (in the case of this package, it is represented as
+// ephemeral key concatenated with the sender's public key) using the recipient's privKey/pubKey keypair,
+// this is equivalent to libsodium's C function: crypto_box_seal()
```

Comment:
It would be good to have some test cases that demonstrate this equivalence.

Reply:
will add a todo to add more tests to ensure interoperability between libsodium and and box.Seal

Reply:
I'm also a bit curious about the equivalence between a "decrypt" function and `crypto_box_seal`